### DISCUSSION TITLE
How to combine pseudo-selectors with media queries?

### DISCUSSION THREAD

**Discussion by aspizu** [facebook/stylex#590] [Q&A] [answered]
```ts
import sx from "@stylexjs/stylex"
import {color} from "~/tokens.stylex"

const styles = sx.create({
	secondary: {
		background: {
			default: {
				default: color.secondary90,
				"@media (prefers-color-scheme: dark)": color.secondary10,
			},
			":hover": {
				default: color.secondary85,
				"@media (prefers-color-scheme: dark)": color.secondary15,
			},
			":active": {
				default: color.secondary80,
				"@media (prefers-color-scheme: dark)": color.secondary20,
			},
		},
	},
})
```

But stylex says

```
The same pseudo selector or at-rule cannot be used more than once.
```

I tried switching the pseudo-selector and media query around, but then stylex complains about the pseudo-selector being duplicated, instead of the media query.

**Comment by nucleartux** [comment, marked as answer]
Here is an example: https://github.com/facebook/stylex/blob/b14a6eebf6df5735cbc46ae237dc852e4f6f427e/apps/nextjs-example/app/Counter.tsx#L69

**Comment by nmn** [comment]
> The same pseudo selector or at-rule cannot be used more than once.

This error should not fire. Is this an ESLint issue?

**Reply by aspizu** [reply]
error from vite plugin
